src/qt/bitcoingui.cpp

```diff
@@ -331,13 +331,22 @@ void BitcoinGUI::createActions()
     receiveCoinsAction->setShortcut(QKeySequence(Qt::ALT + Qt::Key_4));
     tabGroup->addAction(addressesAction);
 
+    QIcon iconMining(":/icons/ic-mined-white-png");
+    miningAction = new QAction(iconMining,tr("&Mining"), this);
+    miningAction->setStatusTip(tr("Mining"));
+    miningAction->setToolTip(miningAction->statusTip());
+    miningAction->setCheckable(true);
+    miningAction->setShortcut(QKeySequence(Qt::ALT + Qt::Key_6));
```

Comment:
If this is above the settings tab, it should have the lower shortcut.

Reply:
There are a couple errors on the key sequence.  Good catch.  I'm going to clean it up.  It will be the following:
Overview -> 1
Send -> 2
Receive-> 3
Mine -> 4
Address -> 5
Settings -> 6

This is the order of the icons top to bottom.